I had to change version format form Maj.Min.Fix to Maj.Min because it is not supported in new sdk:

Build failed
Traceback (most recent call last):
  File "/opt/pebble/Pebble/.waf-1.7.11-04343383b3a9511074383c51ae32d355/waflib/Task.py", line 123, in process
    ret=self.run()
  File "/opt/pebble/Pebble/.waf-1.7.11-04343383b3a9511074383c51ae32d355/waflib/Task.py", line 47, in run
    return m1(self)
  File "/opt/pebble/Pebble/.waf-1.7.11-04343383b3a9511074383c51ae32d355/waflib/extras/pebble_sdk.py", line 58, in _generate_appinfo_c_file_rule
    generate_appinfo.generate_appinfo(task.inputs[0].abspath(),task.outputs[0].abspath())
  File "/opt/pebble/Pebble/.waf-1.7.11-04343383b3a9511074383c51ae32d355/waflib/extras/generate_appinfo.py", line 42, in generate_appinfo
    raise Exception('appinfo.json versionLabel format for app revision must be "Major" or "Major.Minor"')
Exception: appinfo.json versionLabel format for app revision must be "Major" or "Major.Minor"
[ERROR   ] A compilation error occurred

So I'll set it to 1.03, ok?

--- a/appinfo.json
+++ b/appinfo.json
@@ -4,7 +4,7 @@
   "longName": "Acceleromonitor by Seb Maynard",
   "companyName": "seb.so",
   "versionCode": 102,
-  "versionLabel": "1.02",
+  "versionLabel": "1.03",
   "watchapp": {
     "watchface": false
   },

